apps/src/lib/kits/maker/ui/SetupChecklist.jsx

```diff
@@ -93,38 +104,38 @@ export default class SetupChecklist extends Component {
           (isChromeOS() || isChrome()) &&
           !shouldUseWebSerial() &&
           this.detectStep(STATUS_APP_INSTALLED, () =>
-            setupChecker.detectChromeAppInstalled()
+            this.setupChecker.detectChromeAppInstalled()
           )
       )
 
       // Is board plugged in?
       .then(() =>
         this.detectStep(STATUS_BOARD_PLUG, () =>
-          setupChecker.detectBoardPluggedIn()
+          this.setupChecker.detectBoardPluggedIn()
         )
       )
 
       // What type of board is this?
       .then(() => {
-        this.setState({boardTypeDetected: setupChecker.detectBoardType()});
+        this.setState({boardTypeDetected: this.setupChecker.detectBoardType()});
         if (experiments.isEnabled('microbit')) {
-          console.log('Board detected: ' + setupChecker.detectBoardType());
+          console.log('Board detected: ' + this.setupChecker.detectBoardType());
         }
         Promise.resolve();
       })
 
       // Can we talk to the firmware?
       .then(() =>
         this.detectStep(STATUS_BOARD_CONNECT, () =>
-          setupChecker.detectCorrectFirmware(this.state.boardTypeDetected)
+          this.setupChecker.detectCorrectFirmware(this.state.boardTypeDetected)
```

Comment:
Not relevant to this PR, but a thought: I'm not actually sure if we set the board type on the state in the previous `.then` that we're guaranteed it has been updated here (it's asynchronous), but I suppose in reality it hasn't been causing problems?

Reply:
I don't believe this step has been causing problems, but definitely something to note -thanks!